### ISSUE TITLE
How can I make a method called on a new_bar_event async?

### ISSUE THREAD

**Issue by anilwarbhe** [louisnw01/lightweight-charts-python#168]
### Question

`chart.events.new_bar += on_new_bar`

I want to make `on_new_bar` async

```
async def on_new_bar(chart):
     await do_required_processing()
```
How should I go about it? Please suggest.

**Comment by louisnw01** [comment]
Hey @anilwarbhe,

This is not possible with the current version of the library; a workaround would be to use `asyncio.create_task(coro())` inside a sync function subscribed to the new bar event.

Nevertheless, i'll add this feature in a future update.

Louis

**Comment by louisnw01** [comment]
Hey @anilwarbhe,

Fixed in the latest version (1.0.18.2).

Louis
